source/load-balancers/load-balancers.rst

```diff
@@ -116,6 +116,16 @@ the following series of SDAM events:
 Drivers MUST also emit a :code:`ServerClosedEvent` and :code:`TopologyClosedEvent` when
 the topology is closed and MUST NOT emit any other events when operating in this mode.
 
+Log Messages
+^^^^^^^^^^^^
+
+Please refer to the `SDAM logging specification <../server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst#log-messages>`_ 
```

Comment:
may be helpful to also refer users to the monitoring section and point out the same considerations apply for the messages that correspond to the events

Reply:
Sorry I didn't fully understand. Current reference is to the logging section in monitoring.
Do you mean instead of listing the events here just refer to the relevant events?